docs/source/tutorial.rst

```diff
@@ -650,6 +650,43 @@ specific for ML, the BentoML team has also created Yatai and bentoctl:
 Learn more about different deployment options with BentoML from the
 :doc:`concepts/deploy` page.
 
+Use GitHub Action to Boost Your Workflow
+----------------------------------------
+
+BentoML provides a GitHub Action to help you automate the process of building bento and deploying to the cloud.
+Here an example of the workflow file:
+
+.. code-block:: yaml
```

Comment:
I think we should make this a yaml file, and the link it here to rest for us to easier to maintain. As working in yaml file is better than working with yaml within rst